image_tools/README.md

```diff
@@ -0,0 +1,33 @@
+## **What Is This?**
+
+This demo provides a simple utility to **connect to a workstation default camera device** and **display** it in an **OpenCV** window like below:
+
+![](img/result.png)
+
+## **Build**
+
+```bash
+colcon build --package-select image_tools
+```
+
+## **Run**
+
+In `image_tools` ROS 2 package, 2 executables are provided, namely `cam2image` and `showimage` with different functions.
+
+## **1 - cam2image**
+Running this executable connects to your workstation's default camera device's video stream and publishes the images on '/image' and '/flipimage' topics using a ROS 2 publisher.
+
+```bash
+# Open new terminal
+ros2 run image_tools cam2image
+```
+
+## **2 - showimage**
+Running this executable creates a ROS 2 node, `showimage`, which subscribes to the `sensor_msg/msg/Image` topic, `/image` and displays the image messages in an **OpenCV** window.  
```

Comment:
The fact that it is an OpenCV window is an implementation detail (and could very well change in the future).  I think this should just be:

```suggestion
Running this executable creates a ROS 2 node, `showimage`, which subscribes to the `sensor_msg/msg/Image` topic, `/image` and displays the images in a window.  
```

Reply:
Noted. Done.